cmd/prober/prober.go

```diff
@@ -104,42 +108,46 @@ func encodeLevel() zapcore.LevelEncoder {
 }
 
 var (
-	logStyle       string
-	frequency      int
+	scPath      string
+	trPath      string
+	trustedRoot *root.TrustedRoot
```

Comment:
This also doesn't need to be global. These global variables are meant to be assigned to command line flag values, but otherwise it's best to instantiate the variable locally and pass it as an argument to the functions that need it.